package.json

```diff
@@ -61,7 +61,7 @@
     "diff": "5.0.0",
     "escape-string-regexp": "4.0.0",
     "find-up": "5.0.0",
-    "glob": "7.2.0",
+    "glob": "8.1.0",
```

Comment:
glob has published v9*. Is mocha affected by v9 breaking changes, or why are you limiting your upgrade to v8?